expyfun/io/tests/test_parse.py

```diff
@@ -45,3 +46,49 @@ def test_parse():
     data = read_tab(ec.data_fname, group_end=None)
     assert_equal(len(data[0]['misc']), 2)  # includes between-trials stuff
     assert_equal(len(data[1]['misc']), 2)
+
+def test_reconstruct():
+    """Test Tracker objects reconstruction"""
+    from expyfun.stimuli import TrackerUD, TrackerBinom, TrackerDealer
+    from expyfun.io._parse import reconstruct_tracker, reconstruct_dealer
```

Comment:
Is this cheating? To import from `expyfun.io._parse` just seems a little sketchy.